build_omaha.py

```diff
@@ -0,0 +1,152 @@
+  f = open(target_manifest_path,'w')
+  f.write(newdata)
+  f.close()
+
+  # update standalone_installers.txt.
+  installer_text = "('STANDALONE_FILE_NAME', 'STANDALONE_FILE_NAME', [('BRAVE_VERSION', '$STAGING_DIR/BRAVE_INSTALLER_EXE', 'APP_GUID')], None, None, None, False, '', '')"
+  installer_text = installer_text.replace("APP_GUID", args.guid[0])
+  installer_text = installer_text.replace("STANDALONE_FILE_NAME", os.path.splitext(args.standalone_installer_exe[0])[0])
+  installer_text = installer_text.replace("BRAVE_INSTALLER_EXE", args.brave_installer_exe[0])
+  installer_text = installer_text.replace("BRAVE_VERSION", args.brave_full_version[0])
+
+  target_installer_text_path = os.path.join(omaha_dir, 'omaha', 'standalone', 'standalone_installers.txt')
+  f = open(target_installer_text_path,'w')
+  f.write(installer_text)
+  f.close()
+
+def Tagging(args, omaha_dir):
+  opt_win_folder = os.path.join(omaha_dir, 'omaha', 'scons-out', 'opt-win')
+  apply_tag_exe = os.path.join(opt_win_folder, 'obj', 'tools', 'ApplyTag', 'ApplyTag.exe')
+
+  tag = 'appguid=APP_GUID&appname=TAG_APP_NAME&needsadmin=prefers&lang=en&ap=TAG_AP'
+  tag = tag.replace("APP_GUID", args.guid[0])
+  tag = tag.replace("TAG_APP_NAME", args.tag_app_name[0])
+  tag = tag.replace("TAG_AP", args.tag_ap[0])
+
+  source_standalone_installer = os.path.join(opt_win_folder, 'Test_Installers', 'UNOFFICIAL_' + args.standalone_installer_exe[0])
+  target_standalone_installer = os.path.join(args.root_out_dir[0], args.standalone_installer_exe[0])
+  command = [apply_tag_exe, source_standalone_installer, target_standalone_installer, tag]
+  sp.check_call(command, stderr=sp.STDOUT)
+
+  source_stub_installer = os.path.join(opt_win_folder, 'staging', 'BraveUpdateSetup.exe')
+  target_stub_installer = os.path.join(args.root_out_dir[0], args.stub_installer_exe[0])
+  command = [apply_tag_exe, source_stub_installer, target_stub_installer, tag]
+  sp.check_call(command, stderr=sp.STDOUT)
+  return
+
+def ParseArgs():
+  parser = argparse.ArgumentParser(description='build omaha installer')
+  parser.add_argument('--root_out_dir',
+                      nargs=1)
+  parser.add_argument('--brave_installer_exe',
+                      nargs=1)
+  parser.add_argument('--stub_installer_exe',
+                      nargs=1)
+  parser.add_argument('--standalone_installer_exe',
+                      nargs=1)
+  parser.add_argument('--guid',
+                      nargs=1)
+  parser.add_argument('--install_switch',
+                      nargs=1)
+  parser.add_argument('--tag_ap',
+                      nargs=1)
+  parser.add_argument('--tag_app_name',
+                      nargs=1)
+  parser.add_argument('--brave_full_version',
+                      nargs=1)
+  return parser.parse_args()
+
+# Wait to see ths compile logs. If not, elevated python program will quit w/o
+# any notice.
+def WaitFromUser():
+  raw_input('\nPress ENTER to exit.')
+
+def Main(args):
+  ret = run_as_admin()
+  if ret is True:
```

Comment:
This code runs in a separate admin process.
non admin process just returned here. So, `create_dist` process is quit but in a separate process & terminal, omaha build is in-progress.